frontend/src/components/samples/details/SampleDetailsLineage.tsx

```diff
@@ -86,10 +86,10 @@ function SampleDetailsLineage({sample, handleSampleClick, handleProcessClick} :
   const [nodesToEdges, setNodesToEdges] = useState<{ [key: string]: SampleLineageGraphProcessMeasurement }> ({})
   
   useEffect(() => {
-    if (sample.id === undefined) {
-      return;
-    }
     (async () => {
```

Comment:
is there any point in having an async callback inside an empty not async callback ?

Reply:
Well inside `useEffect` is the only appropriate place for executing async function (in a component), and `useEffect` cannot take an async callback as argument, so that pattern was kinda necessary unless we give that async callback a name.